consensus/hotstuff/votecollector/consensus_cluster_vote_collector.go

```diff
@@ -33,17 +31,45 @@ func (c *ConsensusClusterVoteCollector) CreateVote(block *model.Block) (*model.V
 	panic("implement me")
 }
 
+// parseSigType returns type of signature contained in model.Vote.
+func (c *ConsensusClusterVoteCollector) parseSigType(vote *model.Vote) (hotstuff.SigType, error) {
+	panic("implement me")
```

Comment:
Added single sig [encoding & decoding interface here](https://github.com/onflow/flow-go/pull/1140/commits/06deac4f1900c87abf9f8abfc2488d71190f760c#diff-c567bc762113b75ac2bbb6f67b24c844001dd354f8e53b6adc732a6abe9e00c7R12)

This method could use simply use that function

Reply:
yes, I agree, it mainly depends right now on https://github.com/onflow/flow-go/pull/1196